src/cli.js

```diff
@@ -89,12 +90,401 @@ async function cli() {
 		process.exit(result.code);
 	}
 
-	const result = generator(PACKAGES.get);
-	if (result.messages) {
-		result.messages.map((error) => {
-			log.error(error);
-		});
-	}
+	// const result = generator(PACKAGES.get);
+	// if (result.messages) {
+	// 	result.messages.map((error) => {
+	// 		log.error(error);
+	// 	});
+	// }
```

Comment:
un-comment this again?
```suggestion
	const result = generator(PACKAGES.get);
	if (result.messages) {
		result.messages.map((error) => {
			log.error(error);
		});
	}
```